src/LogAnalysis/Types.hs

```diff
@@ -100,20 +107,28 @@ instance Show Knowledge where
 -- | Sorted accoring to knowledgebase.
 -- Tag needs to be in lowercase since Zendesk automatically convert any uppercase
 -- lowercase
+
 renderErrorCode :: ErrorCode -> Text
-renderErrorCode DBError              = "db-corrupted"
-renderErrorCode StaleLockFile        = "stale-lock-file"
+renderErrorCode TLSCert              = "tls-cert-error"
```

Comment:
I still don't understand why you removed some of the existing entries. You said that there weren't any of them in the ticket tags, but that doesn't mean that there won't be any of them.

And it begs the questions where is the knowledge base coming from - from our CSV/YouTrack or your head? If it's coming from your head then that is no good. If you don't have them in YouTrack, please write down (in YouTrack) which ones have been removed and why, since only you know that now.

Reply:
The CSV is generated from YouTrack. i export the CSV, then transform it using the python script I wrote.

Reply:
Re-added the entries I previously removed.